apel_report.py

```diff
@@ -221,10 +230,9 @@ def main():
         year,month = auto_year_month()
     else:
         try:
-            year,month = map(int, sys.argv[1:])
+            year,month = list(map(int, sys.argv[1:]))
```

Comment:
```suggestion
            year,month = map(int, sys.argv[1:])
```

conversion to a list here is unnecessary

Reply:
2to3 made that change.  Odd, right?

Reply:
Eh, presumably `map(...)` -> `list(map(...))` is just a standard conversion ... It is unnecessary in contexts where the return value is consumed as a sequence rather than an actual list, but i don't know how smart 2to3 is at detecting that kind of thing.